assets/i18n/en.json

```diff
@@ -53,7 +53,7 @@
     "dgc-get-dogecoin-how": "Find out how to get Dogecoin.",
     "dgc-stores-title": "Stores.", 
     "dgc-stores-buy": "Buy with Dogecoin.",
-    "dgc-footer-credits": "Created by <a href=\"http://www.ummjackson.com\" target=\"_blank\">Jackson Palmer</a> & <a href=\"https://twitter.com/BillyM2k\" target=\"_blank\">Shibetoshi Nakamoto</a>. Website Design by <a href=\"https://www.reddit.com/user/PowerLemons\" target=\"_blank\">/u/PowerLemons</a>. Logo by <a href=\"http://christinemix.com\" target=\"_blank\">Christine Ricks</a>.",
+    "dgc-footer-credits": "Created by <a href=\"http://www.ummjackson.com\" target=\"_blank\">Jackson Palmer</a> & <a href=\"https://twitter.com/BillyM2k\" target=\"_blank\">Shibetoshi Nakamoto</a>. Website Design by <a href=\"https://www.reddit.com/user/edswf1\" target=\"_blank\">/u/edswf1</a>. Logo by <a href=\"http://christinemix.com\" target=\"_blank\">Christine Ricks</a>.",
```

Comment:
old url `http://www.ummjackson.com` here and in other translation files

Reply:
I saw those, those are also on the current site. There's also a couple more http links in translation files that need to be switched to https. I'll prepare a PR before launch so we can keep this going.

Reply:
> I saw those, those are also on the current site. There's also a couple more http links in translation files that need to be switched to https. I'll prepare a PR before launch so we can keep this going.

sorry missed your comment and updated already